config/v3_4_experimental/translate/translate.go

```diff
@@ -66,12 +66,22 @@ func translateLuks(old old_types.Luks) (ret types.Luks) {
 	return
 }
 
+func translateTang(old old_types.Tang) (ret types.Tang) {
+	tr := translate.NewTranslator()
+	tr.Translate(&old.Thumbprint, &ret.Thumbprint)
+	tr.Translate(&old.URL, &ret.URL)
+	ret.OfflineProvisioning = util.BoolToPtr(false)
+	ret.Advertisement = nil
```

Comment:
These are zero-values or semantically equivalent, so we usually just omit the assignments.

Reply:
wooow, yes; that makes sense.